platforms/nuget/templates/OpenCVNuget.nuspec

```diff
@@ -18,17 +18,16 @@
 		</authors>
 	</metadata>
 	<files>
-		<file src="include\**\*.*" target="build\native\include" />
+		<file src="{{ params.directory }}\include\**\*.*" target="build\native\include" />
 
 		{% for msvc in params.compilers -%}
 		{% for arch in params.architectures -%}
-		<file src="{{ arch }}\{{ msvc }}\lib\**\opencv_world*.lib" target="build\native\{{ arch }}\{{ msvc }}.0\lib" />
-		<file src="{{ arch }}\{{ msvc }}\bin\**\opencv_world*.dll" target="build\native\{{ arch }}\{{ msvc }}.0\bin" />
-		<file src="{{ arch }}\{{ msvc }}\bin\**\opencv_world*.pdb" target="build\native\{{ arch }}\{{ msvc }}.0\bin" />
+		<file src="{{ params.directory }}\{{ arch }}\{{ msvc }}\lib\**\opencv_world*.lib" target="build\native\{{ arch }}\{{ msvc }}.0\lib" />
+		<file src="{{ params.directory }}\{{ arch }}\{{ msvc }}\bin\**\opencv_world*.dll" target="build\native\{{ arch }}\{{ msvc }}.0\bin" />
```

Comment:
In my opinion, the dlls to include in the nuspec should be configurable to enable non-world dll compilations to produce also a nuget.

Reply:
Currently, the build process does not have a dynamic flag to set the options to build world file vs to set the options to build select packages. This is because we would need to parse, or send the list of packages needed and the build process may/may not work with cmake.

For now, the Github Action builds the world lib and dll files on purpose (instead of n individual lib and dll files), since this would be the most useful case for NuGet distribution.